src/routes/users.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Since you are using ES6 features such as const, you can remove the 'use strict' directive as it is not needed in ES6 modules.